DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: secuTrialR
 Type: Package
 Title: Handling of data from the clinical data management system secuTrial 
-Version: 0.8.9
+Version: 0.9.0
```

Comment:
is this not more of an 0.8.10?

Reply:
I was thinking so too, but since so many files are affected I though I would push an 0.9.0

Reply:
fair enough. It's not a big deal either way I don't think